thunderstorm/src/main/frontend/server-info/Button_VersionUpdate.scss

```diff
@@ -0,0 +1,17 @@
+@use '@nu-art/ts-styles' as styles;
+
+.version-update-message {
+  opacity: 1;
+  animation: fadeInOut 6s infinite; // 6 seconds total (4s visible, 1s fade out, 1s fade in)
```

Comment:
I am not sure fading completely out is a good idea.. perhaps it is better to flash it rather then make it disappear

Reply:
Default animation is default.
CSS allows easy overriding.

Reply:
agree but still even infra behaviour can be good behaviour.. it doesn't need to behave bad because it is infra